compiler/modules/front/stmt.bal

```diff
@@ -219,6 +233,7 @@ class StmtContext {
 function codeGenFunction(ModuleSymbols mod, s:FunctionDefn defn, bir:FunctionSignature signature) returns bir:FunctionCode|CodeGenError {
     StmtContext cx = new(mod, defn, signature.returnType);
     bir:BasicBlock startBlock = cx.createBasicBlock();
+    cx.openRegion(startBlock.label, bir:REGION_SIMPLE);
```

Comment:
We don't need this region.

Reply:
Can you have a look at this 
https://github.com/poorna2152/nballerina/issues/14#issuecomment-1070707268